src/components/AppSelect.vue

```diff
@@ -34,6 +38,10 @@ export default {
       type: Boolean,
       default: false,
     },
+    errors: {
```

Comment:
This prop is actually an error message, it is not plural?
so name it more appropriately